.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: android
+android:
+  components:
```

Comment:
To use the latest SDK Platform Tools add the following lines described [here](https://docs.travis-ci.com/user/languages/android/#Installing-a-newer-SDK-Platform-Tools-revision).